examples/buildings/IfcOpenShellLoaderPlugin_Duplex.html

```diff
@@ -0,0 +1,225 @@
+           target="_other">IFCOpenShellLoaderPlugin</a> is the easiest way to load IFC models into a xeokit Viewer.</p>
+    <p>IFCOpenShellLoaderPlugin loads IFC STEP files and parses them within the browser using <a
+            href="https://docs.ifcopenshell.org/"
+            target="_other">ifcopenshell.api</a>, to create 3D objects within the
+        Viewer.</p>
+    <h3>Limitations</h3>
+    <p>Loading and parsing huge IFC STEP files can be slow, and can overwhelm the browser, however. To view your
+        largest IFC models, we recommend instead pre-converting those to xeokit's compressed native .XKT format, then
+        loading them with <a href="../../docs/class/src/plugins/XKTLoaderPlugin/XKTLoaderPlugin.js~XKTLoaderPlugin.html"
+                             target="_other">XKTLoaderPlugin</a>.</p>
+    <h3>Stats</h3>
+    <ul>
+        <li>
+            <div id="time">Loading JavaScript modules...</div>
+        </li>
+    </ul>
+    <h3>Components used</h3>
+    <ul>
+        <li>
+            <a href="../../docs/class/src/viewer/Viewer.js~Viewer.html"
+               target="_other">Viewer</a>
+        </li>
+        <li>
+            <a href="../../docs/class/src/plugins/IFCOpenShellLoaderPlugin/IFCOpenShellLoaderPlugin.js~IFCOpenShellLoaderPlugin.html"
+               target="_other">IFCOpenShellLoaderPlugin</a>
+        </li>
+        <li>
+            <a href="../../docs/class/src/plugins/TreeViewPlugin/TreeViewPlugin.js~TreeViewPlugin.html"
+               target="_other">TreeViewPlugin</a>
+        </li>
+        <li>
+            <a href="../../docs/class/src/plugins/NavCubePlugin/NavCubePlugin.js~NavCubePlugin.html"
+               target="_other">NavCubePlugin</a>
+        </li>
+    </ul>
+    <h3>Assets</h3>
+    <ul>
+        <li>
+            <a href="http://openifcmodel.cs.auckland.ac.nz/Model/Details/274"
+               target="_other">Model source</a>
+        </li>
+    </ul>
+</div>
+</body>
+
+<script type="module">
+
+    import {Viewer, IFCOpenShellLoaderPlugin, NavCubePlugin, TreeViewPlugin} from "../../dist/xeokit-sdk.es.js";
+
+    const viewer = new Viewer({
+        canvasId: "myCanvas",
+        transparent: true
+    });
+
+    viewer.camera.eye = [-3.933, 2.855, 27.018];
+    viewer.camera.look = [4.400, 3.724, 8.899];
+    viewer.camera.up = [-0.018, 0.999, 0.039];
+
+    new NavCubePlugin(viewer, {
+        canvasId: "myNavCubeCanvas",
+        visible: true,
+        size: 250,
+        alignment: "bottomRight",
+        bottomMargin: 100,
+        rightMargin: 10
+    });
+
+    new TreeViewPlugin(viewer, {
+        containerElement: document.getElementById("treeViewContainer"),
+        autoExpandDepth: 3 // Initially expand the root tree node
+    });
+
+    const ifcLoader = new IFCOpenShellLoaderPlugin(viewer, {
+        workerSrc: "./../dist/IFCOpenShellWorker.js",
```

Comment:
Michał discovered we needed to change this to "../../../dist/IFCOpenShellWorker.js" to make it work